core/ledger/blockchain.go

```diff
@@ -17,6 +17,7 @@ type Blockchain struct {
 	BlockPersistTime map[Uint256]int64
 	BCEvents         *events.Event
 	mutex            sync.Mutex
+	mutex2           sync.Mutex
```

Comment:
A bit confuse as there are two mutex in one data structure, the purpose of the mutex need be explain in the comments.